src/lib.rs.in

```diff
@@ -140,31 +142,43 @@ macro_rules! es_body_op {
 
 impl Client {
     /// Create a new client
-    pub fn new(host: &str, port: u32) -> Client {
+    pub fn new(host: &str) -> Client {
+        let mut url = Url::parse(host).unwrap();
```

Comment:
I'm not sure about the `unwrap` for something this deep.  It would probably be better to change the signature of the function to return `Result<Client, SomeTypeOfError>` instead.  At least that gives the calling function a choice on what to do next.